tests/qam-minimal/install_patterns.pm

```diff
@@ -25,14 +25,27 @@ use utils;
 use qam;
 use testapi;
 
+sub install_packages {
+    my $patch_info = shift;
+    my $pattern    = qr/\s+(.+)(?!\.src)\..*\s<\s.*/;
+
+    # loop over packages in patchinfo and try instalation
+    foreach my $line (split(/\n/, $patch_info)) {
+        if (my ($package) = $line =~ $pattern) {
+            script_run("zypper -n in $package", 700);
+            save_screenshot;
+        }
+    }
+}
+
 sub run {
     my ($self) = @_;
+    my $patch = get_var('INCIDENT_PATCH');
```

Comment:
If they are not needed, then for the sake of consistency I would suggest to treat all the `settings` in the same way. So, either remove the check for both, or add it for both ;)